### ISSUE TITLE
Talk service seems to not exist

### ISSUE THREAD

**Issue by ivankudibal** [influxdata/docs-v2#3861]
Talk service seems to not exist

##### Relevant URLs
- https://docs.influxdata.com/kapacitor/v1.6/event_handlers/talk/
- _Provide other relevant URLs (project issues, community threads, etc.)_

Documentation talks about Talk as a provider. If you open the Talk page, there is an empty page w/o response.

Google search provides answer, that Talk was renamed to [Team ambition](http://www.teambition.com/)

Teambition cannot be downloaded as an app properly and cannot be used as long as user does not have Koeran/Japenese/Chinese telephone number...

The question is should we keeep or remove Talk from supported alert systems... Reason for removal is because we cannot test it.

**Comment by lwandzura** [comment]
I think this will need to be removed, will confirm with @docmerlin

**Comment by docmerlin** [comment]
I don't know enough about this @samhld do you know more?

**Comment by samhld** [comment]
@docmerlin never heard of it -- will look into it

**Comment by lwandzura** [comment]
@samhld I believe this is now called Teambition - https://www.teambition.com/

**Comment by lwandzura** [comment]
@samhld Any updates on this?  I'd like to remove from Docs if no longer supported.

**Comment by kelseiv** [comment]
@lwandzura, yes, let's remove the page in the current version and one version back (from the nav menu too), and then:
add an alias to the Event Handler page: https://docs.influxdata.com/kapacitor/v1.6/event_handlers/ to be sure if anyone has that Talk page bookmarked or cached, they'll be re-directed.

On the Event Handler page, let's leave the mention of Talk in the bullet list, and add (Deprecated) after it

**Comment by lwandzura** [comment]
closed via https://github.com/influxdata/docs-v2/pull/4379, forgot to link
